source/transactions/transactions.rst

```diff
@@ -1373,6 +1380,7 @@ durable, which achieves the primary objective of avoiding duplicate commits.
 **Changelog**
 -------------
 
+:2019-06-10: Clarified client side error handling.
```

Comment:
Alternative suggestion: "Client side errors must not change transaction state". Not much longer, and it captures the entire change.